src/mscorlib/src/System/Resources/IResourceManager.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Globalization;
+using System.Runtime.CompilerServices;
+
+namespace System.Resources
+{
+    /// <summary>
+    /// Private interface between CoreLib and ResourceManager so that CoreLib can access resources
```

Comment:
My testing on CoreFX shows I generally don't need to do that (I believe because this layering already exists on ProjectN). I did need to switch System.Collections to use the existing SR.Core indirection we use elsewehre, but the new build system may make that unnecessary..